src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -179,7 +179,7 @@ asset::SAssetBundle CImageLoaderTGA::loadAsset(io::IReadFile* _file, const asset
 	imgInfo.type = ICPUImage::ET_2D;
 	imgInfo.extent.width = header.ImageWidth;
 	imgInfo.extent.height = header.ImageHeight;
-	imgInfo.extent.depth = 1u;
+	imgInfo.extent.depth = header.PixelDepth / 8; // not sure about it
```

Comment:
no depth as in dimension size depth is 1u (image is always 2D)

PixelDepth from TGA header probably means how many bits per pixel (color depth), so its used for picking the correct E_FORMAT